extensions/amp-video/amp-video.md

```diff
@@ -80,13 +84,9 @@ Alternatively, you can present a click-to-play overlay. For details, see the [Cl
 
 ##### autoplay
 
-If this attribute is present, and the browser supports autoplay:
-
-* the video is automatically muted before autoplay starts
-* when the video is scrolled out of view, the video is paused
-* when the video is scrolled into view, the video resumes playback
-* when the user taps the video, the video is unmuted
-* if the user has interacted with the video (e.g., mutes/unmutes, pauses/resumes, etc.), and the video is scrolled in or out of view, the state of the video remains as how the user left it.  For example, if the user pauses the video, then scrolls the video out of view and returns to the video, the video is still paused.
+If this attribute is present, and the browser supports autoplay, the video will be automatically
+played as soon as it becomes visible. There are some conditions that the component needs to meet
+to be played, [which are outlined here](https://github.com/ampproject/amphtml/blob/master/spec/amp-video-interface.md#autoplay).
```

Comment:
Instead of here, provide doc title, like: `which are outlined in the [Video in AMP spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-video-interface.md#autoplay).`